Library/Homebrew/cask/audit.rb

```diff
@@ -582,8 +582,9 @@ def check_livecheck_min_os
 
       return if cask_min_os == min_os_string
 
+      min_os_symbol = cask_min_os&.to_sym.inspect || "no minimal OS version"
```

Comment:
@Rylan12 I think that adding the `&` in the first half means that `min_os_symbol` becomes "nil", instead of the nicer error message.

Reply:
Oops, good catch 😅

There's a fix in #14302